1713000002937	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1713000141290	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000149279	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000150507	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000152601	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000156195	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000164275	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000165931	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000198893	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1713000604212	Sync.Service	DEBUG	User-Agent: LibreWolf/124.0.1 (Linux x86_64) FxSync/1.126.0.20240321230221.desktop
1713000604213	Sync.Service	INFO	Starting sync at 2024-04-13 11:30:04 in browser session YYQHhDZxWf3d
1713000604213	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1713000604215	Sync.Status	INFO	Resetting Status.
1713000604215	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1713000604216	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1713000604216	FirefoxAccounts	DEBUG	getOAuthToken enter
1713000604216	FirefoxAccounts	TRACE	getCachedToken returning cached token
1713000604216	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1713000604216	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1713000644257	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1713000644257	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1713000644577	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/206026008/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1713000644577	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1713000644577	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1713000644577	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1713000644578	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1713000644579	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
